### ISSUE TITLE
Sort extra files list

### ISSUE THREAD

**Issue by Tech356** [schubergphilis/chef-guard#106]
When I upload a cookbook with extra files Chef guard gives a list of the extra files. I would be nice if this list was sorted so it is easier to read and to find the offending files in the filesystem.

Here is an attempted upload of the Windows cookbook after I cloned the git repo. 

```
c:\chef-repo>knife cookbook upload windows --freeze
Uploading windows        [1.39.0]
ERROR: Precondition Failed
Response:
=== Cookbook Compare errors found ===
Your upload contains more files than the source cookbook:
 - chefignore
 - Gemfile
 - .foodcritic
 - MAINTAINERS.md
 - MAINTAINERS.toml
 - Rakefile
 - LICENSE
 - .kitchen.yml
 - .rubocop.yml
 - TESTING.md
 - TODO.md
 - .kitchen.cloud.yml
 - CONTRIBUTING.md

Source: https://supermarket.chef.io/api/v1/cookbooks/windows/versions/1.39.0/download
=====================================
```

It would be better if the list was sorted

```
c:\chef-repo>knife cookbook upload windows --freeze
Uploading windows        [1.39.0]
ERROR: Precondition Failed
Response:
=== Cookbook Compare errors found ===
Your upload contains more files than the source cookbook:
 - .foodcritic
 - .kitchen.cloud.yml
 - .kitchen.yml
 - .rubocop.yml
 - chefignore
 - CONTRIBUTING.md
 - Gemfile
 - LICENSE
 - MAINTAINERS.md
 - MAINTAINERS.toml
 - Rakefile
 - TESTING.md
 - TODO.md

Source: https://supermarket.chef.io/api/v1/cookbooks/windows/versions/1.39.0/download
=====================================
```

**Comment by svanharmelen** [comment]
Should be easy enough, will have a look at it...

**Comment by svanharmelen** [comment]
Fixed by PR #108 which is released in v0.7.0
